### ISSUE TITLE
Import map from a JSR specifier with path can't be resolved

### ISSUE THREAD

**Issue by hasundue** [hasundue/molt#146]
For example, running molt CLI for a TS file

```ts
import { describe } from "@std/testing/bdd";
```

along with deno.json 

```json
{
  "imports": {
    "@std/testing": "jsr:@std/testing@0.210.0"
  }
}
```

results in the following error:

```
AssertionError: Expected actual: "undefined" to not be null or undefined.
```

Now being fixed in #145

**Comment by hasundue** [comment]
You can workaround this by writing like

```json
{
  "imports": {
    "@std/testing/bdd": "jsr:@std/testing@0.210.0/bdd"
  }
}
```

instead, but it's not as handy as the failing example.

**Comment by hasundue** [comment]
You can also avoid this error by running the CLI against `deno.json` directly.

**Comment by hasundue** [comment]
Seems like a bug in the import_map crate. I created an issue and maybe working on it.
https://github.com/denoland/import_map/issues/78

**Comment by hasundue** [comment]
#166 introduced a tentative fix on this, which makes molt print warning messages and continue, rather than crushing with an unfriendly error.

I believe I can fix this in this week, so please be patient for the inconvenience.

**Comment by hasundue** [comment]
Should be fixed in 0.18.3
